config/routes.rb

```diff
@@ -3,7 +3,11 @@
 Jets.application.routes.draw do
   namespace :api do
     resources :agencies, only: :index
-    resources :event_dates, only: :show
+    resources :event_hours, only: %i[index show]
```

Comment:
What's the purpose of having `/event_hours` and `/event_date/:id/event_hours`? Do we really need both?

Reply:
Not sure how this plays out in the API, but from a business case I think it makes sense to be able to retrieve all event hours (and connected event_slots) from a single event_date_id.